.travis.yml

```diff
@@ -28,8 +28,8 @@ matrix:
           env:  DB=pgsql CORE_BRANCH=stable13
         - php: 7.1
           env:  DB=pgsql CORE_BRANCH=stable13
-        - php: 7.2
-          env:  DB=pgsql CORE_BRANCH=stable13
+#        - php: 7.2
+#          env:  DB=pgsql CORE_BRANCH=stable13
```

Comment:
Why did you have to remove this?
Any idea how we could solve it?

Reply:
I removed it for now . It's a known phpunit issue, I tried to solve it earlier, But my main focus was to have working tests again :)

We'll need to push a fix later on :)
https://github.com/sebastianbergmann/phpunit/issues/2453